packages/hardhat/hardhat.config.js

```diff
@@ -179,7 +179,6 @@ module.exports = {
     },
     mainnetAvalanche: {
       url: "https://api.avax.network/ext/bc/C/rpc",
-      gasPrice: 225000000000,
```

Comment:
Hardcoded gas price?

Reply:
Should the gasPrice attribute be added back in?

Reply:
Hardcoded gas price was removed for me to be able to deploy the smart contracts. Otherwise, it wouldn't work

Reply:
@varunv5000

Reply:
@farque65  Yeah, it could probs be added back in to keep in line with scaffold

Reply:
@varunv5000 Sounds good